index.js

```diff
@@ -1,10 +1,10 @@
 require('construct-style-sheets-polyfill');
 
-SystemJS.import('./main.css').then(function (module) {
+System.import('./main.css').then(function (module) {
```

Comment:
You can do `System.import()` and not have it be a code split, if you make the proper changes to your webpack config. See https://github.com/systemjs/systemjs#compatibility-with-webpack